### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bherrmann7**
minor typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ The admin api only allows the creating of users using the email/password sign-in
 ```
 
 ### Working with Firestore (Alpha)
-The the API allows for CRUB operation on firestore.
+The the API allows for CRUD operation on firestore.
```
Comment (alekcz) [inline README.md]: @bherrmann7 could you also fix the duplicate the at the beginning of the sentence. Then I'll do a merge.

**Comment by bherrmann7** [comment]
Ha! funny that I saw the CRUD but not the "the the"

Cheers!
-bob

**Review by alekcz — APPROVED** [review]
Thanks so much for the contribution. Much appreciated.

**Comment by bherrmann7** [comment]
I enjoyed watching your talk from the conj!

**Pull request merged**